rotkehlchen/tests/api/test_manually_tracked_balances.py

```diff
@@ -140,13 +141,19 @@ def _populate_initial_balances(api_server) -> List[Dict[str, Any]]:
     response = requests.put(
         api_url_for(
             api_server,
-            'manuallytrackedbalancesresource',
+            "manuallytrackedbalancesresource",
```

Comment:
Hmm ... no. Why are you changing this to `"`? We use `'` everywhere in rotki.
```suggestion
            'manuallytrackedbalancesresource',
```